themes/socialbase/templates/profile/profile--compact.html.twig

```diff
@@ -20,7 +20,14 @@
  */
 #}
 <div class="avatar">
-  <a href="{{ profile_stream_url }}" title="{{ profile_name|render }}">
-    {{ content.field_profile_image }}
+  <a href="{{ profile_stream_url }}" title="{{ profile_name }}">
+
+    {% if profile_image_url %}
+      <img src="{{ profile_image_url }}" class="img-responsive" alt="{{ profile_name }}">
+    {% else %}
+      {# if default_image #}
+      {{ node.field_profile_image }}
```

Comment:
@left23 you moved from `{{ content.field_profile_image }}` to `{{ node.field_profile_image }}` here. Is that intentional?

Reply:
No, I was experimenting and didn't roll back. Thanks for spotting this. I've removed the trans for profile name in title attribute too.